### ISSUE TITLE
[0.16.8 Crash] Error when using Recipe calculator 

### ISSUE THREAD

**Issue by jarquafelmu** [ElmarWueest/Recexplo#10]
### Description

Game crash on using recipe calculator to select an object.

---------------------------------------------------
### Error
```
Error while running event RecExplo::on_gui_click (ID 1)
__RecExplo__/lua/recipe_cal.lua:127: bad argument #1 to 'len' (string expected, got nil)
```
---------------------------------------------------
### Reproduction

1. Open recipe explorer
1. Select Fast Underground Belt 
1. Open Recipe Calculator
1. Click record
1. Click on Fast Underground Belt in the product section
1. Crash
---------------------------------------------------
### Versions
Factorio 0.16.51
Recexplo 0.16.8

**Comment by jarquafelmu** [comment]
Clicking on any of the ingredient / recipe icons in the recipe explorer crash the game

**Comment by ntman4real** [comment]
+1 same here.

**Comment by ghost** [comment]
+1 Can confirm, just got this as well.

**Comment by jarquafelmu** [comment]
Any news Elmar?

**Comment by ntman4real** [comment]
Bueller?

**Comment by jarquafelmu** [comment]
I think Elmar is ded

**Comment by ElmarWueest** [comment]
no
I was on holidays and had no internet.
will try to fix it today.

**Comment by ElmarWueest** [comment]
fixed in 0.16.9

**Comment by jarquafelmu** [comment]
yay!
